.github/workflows/preview.yml

```diff
@@ -0,0 +1,24 @@
+name: preview-pipeline
+
+on: [push]
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check HTML
+      uses: chabad360/htmlproofer@master
+      with:
+        directory: "."
+    - name: Deploy preview site
+    - uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --exclude '*' --include '*.html' --include 'assets/*' --include 'global'
```

Comment:
These are documented here: https://docs.aws.amazon.com/cli/latest/reference/s3/index.html#use-of-exclude-and-include-filters